includes/controls/groups/background.php

```diff
@@ -258,6 +258,19 @@ public function init_fields() {
 			],
 		];
 
+		$fields['dimensions'] = [
+			'label' => esc_html_x( 'Dimensions', 'Background Control', 'elementor' ),
+			'type' => Controls_Manager::SELECT,
+			'options' => $this->get_images_sizes_options(),
```

Comment:
```suggestion
			'options' => $this->get_image_sizes_options(),
```

Reply:
Not relevant anymore, the logic has changed.